frontend/pages/beta/model/[modelId]/release/new.tsx

```diff
@@ -19,71 +19,69 @@ export default function NewRelease() {
   const [semver, setSemver] = useState('')
   const [releaseNotes, setReleaseNotes] = useState('')
   const [isMinorRelease, setIsMinorRelease] = useState(false)
-  const [artefacts, setArtefacts] = useState<File[]>([])
-  const [artefactsMetadata, setArtefactsMetadata] = useState<FileWithMetadata[]>([])
+  const [files, setFiles] = useState<File[]>([])
+  const [filesMetadata, setFilesMetadata] = useState<FileWithMetadata[]>([])
   const [imageList, setImageList] = useState<FlattenedModelImage[]>([])
   const [errorMessage, setErrorMessage] = useState('')
   const [loading, setLoading] = useState(false)
   const router = useRouter()
 
   const { modelId }: { modelId?: string } = router.query
   const { model, isModelLoading, isModelError } = useGetModel(modelId)
-  const { mutateReleases } = useGetReleasesForModelId(modelId)
 
   const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
     event.preventDefault()
-    if (model) {
-      setErrorMessage('')
-      setLoading(true)
-      if (!model.card.version) {
-        setLoading(false)
-        return setErrorMessage('Please make sure your model has a schema set before drafting a release.')
-      }
-      if (isValidSemver(semver)) {
-        const fileIds: string[] = []
-        for (const artefact of artefacts) {
-          const artefactMetadata = artefactsMetadata.find((metadata) => metadata.fileName === artefact.name)
-          const postArtefactResponse = await postFile(
-            artefact,
-            model.id,
-            artefact.name,
-            artefact.type,
-            artefactMetadata?.metadata,
-          )
-          if (postArtefactResponse.ok) {
-            const res = await postArtefactResponse.json()
-            fileIds.push(res.file._id)
-          } else {
-            setLoading(false)
-            return setErrorMessage(await getErrorMessage(postArtefactResponse))
-          }
-        }
 
-        setLoading(false)
+    if (!model) {
+      return
```

Comment:
This isn't something that we should ever reach, but if we do we should show an error message that encourages users to report what is most definitely a bug (I know we'd missed this already with the old logic). Something like "Failed to submit, model not found. Please contact Bailo support"